Buttler.Backend/Buttler.Domain/Models/User.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Buttler.Domain.Model;
+
+public class User
+{
+    [Key]
+    public int userID { get; set; }
```

Comment:
These should be uppercases since they are public, but good use of the annotations :)